iina/Preference.swift

```diff
@@ -372,6 +370,24 @@ struct Preference {
     }
   }
 
+  enum HardwareDecoderOption: Int {
```

Comment:
It's bound to `selectedTag` of the pop up button. The same for all similar enums in `Preference`.